tracpro/static/js/poll-charts.js

```diff
@@ -39,6 +39,62 @@ jQuery.fn.extend({
             });
         }
     },
+    chart_numeric_split: function() {
+        var dataType = $('#id_numeric').val(); // sum, average or response-rate
+        if (["sum", "average", "response-rate"].indexOf(dataType) != -1) {
+            var label = $('#id_numeric :selected').text();
+            $(this).each(function(i, item) {
+                var chart = $(item);
+                chart.closest('.poll-question').find('.data-type').text(label);
+                var data = chart.data('chart');
+                var seriesData = [];
+                for (i = 0; i < data['region-list'].length; i++) {
+                    seriesData.push({
+                        name: data['region-list'][i],
+                        data: data[dataType][i]
+                    });
+                }
+                chart.highcharts({
+                    chart: {
+                        type: 'area'
+                    },
+                    title: {
+                        text: 'Stacked area chart'
```

Comment:
This isn't a very specific title, and none of the other charts have `title` values passed in.